lib/oath/views/add_account_page.dart

```diff
@@ -453,6 +453,7 @@ class _OathAddAccountPageState extends ConsumerState<OathAddAccountPage> {
                       key: keys.secretField,
                       controller: _secretController,
                       obscureText: _isObscure,
+                      autofillHints: isAndroid ? [] : const [AutofillHints.password],
```

Comment:
This took me a while to realize why it was only for non-Android. Maybe adding a comment would make it more clear.